eng/tools/apidiff/go_mod_tidy_hack.go

```diff
@@ -1,13 +0,0 @@
-// +build modhack
```

Comment:
Removing these might be problematic.  Unfortunately, it's also hard to properly test.